roles/rhods/rhods_deploy_ods/templates/catalog.yml.j2

```diff
@@ -2,11 +2,13 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
   name: {{ rhods_catalog_name }}
-  namespace: {{ rhods_namespace }}
+  namespace: openshift-marketplace
```

Comment:
not sure why you're changing that?
using `{{ rhods_namespace }}` [does works](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/819/pull-ci-openshift-psap-ci-artifacts-main-notebooks-light/1677042016862605312/artifacts/light/prepare/artifacts/003__sutest_rhods__deploy_ods/src/000_rhods_catalog.yml), did you face any problem with it? 🤔

Reply:
I think this made [your test fail](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/835/pull-ci-openshift-psap-ci-artifacts-main-lightspeed-load-test/1677288775329255424/artifacts/load-test/prepare/artifacts/000__rhods__deploy_ods/_ansible.log):
```
2023-07-07 12:49:43,117 p=218 u=psap-ci-runner n=ansible | roles/cluster/cluster_deploy_operator/tasks/main.yml:42
2023-07-07 12:49:43,117 p=218 u=psap-ci-runner n=ansible | TASK: cluster_deploy_operator : Ensure that the CatalogSource exists
2023-07-07 12:49:43,117 p=218 u=psap-ci-runner n=ansible | ----- FAILED ----
2023-07-07 12:49:43,117 p=218 u=psap-ci-runner n=ansible | msg: non-zero return code
2023-07-07 12:49:43,117 p=218 u=psap-ci-runner n=ansible | 
2023-07-07 12:49:43,117 p=218 u=psap-ci-runner n=ansible | <command> oc get CatalogSource/addon-managed-odh-catalog -n redhat-ods-operator
2023-07-07 12:49:43,117 p=218 u=psap-ci-runner n=ansible | 
2023-07-07 12:49:43,117 p=218 u=psap-ci-runner n=ansible | <stderr> Error from server (NotFound): catalogsources.operators.coreos.com "addon-managed-odh-catalog" not found
2023-07-07 12:49:43,117 p=218 u=psap-ci-runner n=ansible | ----- FAILED ----
```
==> ansible is looking for the catalog in the `redhat-ods-operator` namespace, but you created it in `openshift-marketplace`

Reply:
Strange as I think this worked for me locally. The reason that I changed this was that on pre-released OCP versions like 4.13.0-nightly, NFD is not available in the default catalog but it was available in the custom catalog created by this RHODS deploy step so I thought by changing the location of the custom catalog we could also enable installing NFD from the custom catalog in OCP versions where it is not yet available.

I also thought it made more sense to have the catalog in a totally separate namespace from the operator itself.   I will change it back for now.

Reply:
I changed it specifically for RHODS in this PR: #623 , but I don't recall what was behind it
you might be able to deploy NFD from this catalog though, if you use `cluster deploy_operator`, you should be able to customize all the parameters to indicate where to find the resources